src/main/resources/assets/wurst/lang/zh_tw.json

```diff
@@ -1,23 +1,148 @@
-    "description.wurst.hack.autosoup": "當你的生命值低時，將會自動吃湯。\n注意：這個Hack忽略了飢餓值，並假設吃湯會直接補充你的生命值。如果你所玩的伺服器沒有設定成這樣，請使用AutoEat(自動食用)代替。"
+  "description.wurst.hack.anchoraura": "自動放置（可選），充能並引爆重生錨並擊殺你附近的實體。",
+  "description.wurst.hack.antiafk": "隨機走動，用於避免服務器掛機檢測。\n需要至少有3x3的區域可以走動。",
+  "description.wurst.hack.antiblind": "防止失明。\n與OptiFine（高清修復）不相容。",
+  "description.wurst.hack.anticactus": "保護你免受仙人掌傷害。",
+  "description.wurst.hack.antiknockback": "保護你不會被其他生物或玩家推動和用劍擊退。",
+  "description.wurst.hack.antispam": "預防存在的重複刷屏，改為用計數器顯示。",
+  "description.wurst.hack.antiwaterpush": "防止你被水流推動。",
+  "description.wurst.hack.antiwobble": "關閉因反胃和傳送門引起的搖晃效果。",
+  "description.wurst.hack.autoarmor": "自動管理你的盔甲。",
+  "description.wurst.hack.autobuild": "自動建築東西。\n放一個方塊就開始自動建造。",
+  "description.wurst.hack.autodrop": "自動丟棄你不想要的物品（有預設）。",
+  "description.wurst.hack.autoleave": "如果你的血量低於設定值，將會自動退出服務器。",
+  "description.wurst.hack.autoeat": "當必要的時候將會自動進食。",
+  "description.wurst.hack.autofarm": "自動收穫與種植農作物。\n有效作物為小麥、胡蘿蔔、馬鈴薯、紅菜頭根、南瓜、西瓜、仙人掌、甘蔗、海帶、竹子、地獄疣和可可豆。",
+  "description.wurst.hack.autofish": "自動使用最好的魚竿釣魚，如果在此期間獲得了更好的魚竿，就會自動切換到它。",
+  "description.wurst.hack.automine": "自動挖掘你眼前所指的方塊。",
+  "description.wurst.hack.autopotion": "當你的生命值低於設定值時，自動扔一瓶噴濺型瞬間恢復藥水。",
+  "description.wurst.hack.autoreconnect": "當你被服務器踢出時候，自動重連。",
+  "description.wurst.hack.autorespawn": "當你死亡的時候自動重生。",
+  "description.wurst.hack.autosign": "立即在每個牌子上寫下任何你想要的文字，當此功能啟動的時候，你只需要很普通的在第一個牌子上，寫你想要的文字，之後接下來所有牌子都跟上一個一樣。",
+  "description.wurst.hack.autosoup": "當你生命值低下的時候自動喝蘑菇煲。\n\n§l注意：§r這個功能是忽視“饑餓值”，因為特定服務器小遊戲內（如Hypixel），喝湯是直接補充生命，而不是回復飽腹值。如果你正在玩的服務器沒有這個功能，請使用AutoEat功能而不是這個。",
+  "description.wurst.hack.autosprint": "讓你自動進入疾跑。",
+  "description.wurst.hack.autosteal": "你所打開的箱子以及潜影盒會自動盜竊裡面所有物品。",
+  "description.wurst.hack.autoswim": "當你在水中會自動切換泳姿狀態。",
+  "description.wurst.hack.autoswitch": "按順序自動切換快捷欄位。\n\n§l進階提示：§r倘若在你的快捷欄有不同顏色的羊毛以及，不同顏色的混粘土，利用這個功能與BuildRandom同時使用會出現意想不到的效果。",
+  "description.wurst.hack.autosword": "當你攻擊實體的時候，自動使用快捷欄中最好的武器。\n提示：在使用Killaura功能時候，最好開啟。",
+  "description.wurst.hack.autotool": "當你破壞方塊的時候會在你的物品快捷欄中切換成最優的破壞工具。",
+  "description.wurst.hack.autototem": "自動將不死圖騰移動到副手上。",
+  "description.wurst.hack.autowalk": "讓你自動走路，不需要自己按著W鍵。",
+  "description.wurst.hack.basefinder": "通過尋找人為建造的方塊以找到玩家的基地\n假如方塊被找到將會高亮以你所選的顏色。\n建議用於尋找幫派的基地。",
+  "description.wurst.hack.blink": "暫停所有動作更新，並在關閉時繼續。",
+  "description.wurst.hack.boatfly": "允許玩家在使用船飛天。",
+  "description.wurst.hack.bonemealaura": "在指定的作物上自動使用骨粉。\n使用核取方塊來指定作物。",
+  "description.wurst.hack.bowaimbot": "當你用弓或者弩時自動瞄準所設定的實體。",
+  "description.wurst.hack.buildrandom": "在你的附近隨機放置方塊。",
+  "description.wurst.hack.bunnyhop": "讓你自動跳躍，但不能做到像CSGO一樣。",
+  "description.wurst.hack.cameranoclip": "允許第三人稱攝像機忽視牆體碰撞直接把視角穿過去。",
+  "description.wurst.hack.cavefinder": "幫助你找到洞穴並以你所選的顏色將他們高亮。",
+  "description.wurst.hack.chattranslator": "從聊天欄的資訊利用Google翻譯成你所指定的語言（由於一些原因在中國大陸内無法使用，除非你知道如何繞過）。",
```

Comment:
The second half of the narrative can be omitted because the speakers of traditional Chinese do not come from China.
("由於一些原因在中國大陸内無法使用，除非你知道如何繞過")

Reply:
> The second half of the narrative can be omitted because the speakers of traditional Chinese do not come from China. ("由於一些原因在中國大陸内無法使用，除非你知道如何繞過")

I agree with what you talk.